.changeset/popular-glasses-float.md

```diff
@@ -0,0 +1,2 @@
+---
+---
```

Comment:
For changes in the docs, the changeset is not required. You can remove this file.

Reply:
empty changeset is also fine, as mentioned in contributing guide.

Reply:
Ah, okay. Sorry for the noise

Reply:
any way thanks @macci001